app/controllers/projects_controller.rb

```diff
@@ -9,6 +9,8 @@ def index
     @show_filters = true
     @show_search_bar = true
 
+    sort = params[:sort] ? "#{sanitize_sort_column(params[:sort])} #{sanitize_sort_direction(params[:direction])}" : 'highlight DESC, COUNT(volunteers.id) DESC, created_at DESC'
```

Comment:
Instead of doing this can you have params[:sort] point to a key of predefined sort values?

Reply:
eg

```
sort_options = { name_up: ‘name ASC, created_at DESC’, name_down: ‘name DESC, created_at DESC’ }
```